collector/collect.sh

```diff
@@ -13,6 +13,10 @@ while : ; do
         # Update and rebuild the collector.
         git pull
         git reset --hard @{upstream}
+
+        # Make sure we have a recent build, so that we can successfully build
+        # the collector.
+        rustup update
```

Comment:
I wonder if this is "safe", since if the update happens e.g. everyday, we can get an invalid nightly that breaks something? Is nightly even required for the collector, can't we just build it with stable and update that from time to time?

Reply:
Nightly shouldn't break, and if the build still fails after this change we should stay up, just with a stale version (like we did before I ran the update). Moving to stable seems ok; IIRC there was some reason for using nightly here but it might not apply anymore.